shell/platform/windows/compositor_opengl.cc

```diff
@@ -58,17 +45,18 @@ bool CompositorOpenGL::CreateBackingStore(
   gl_->TexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_NEAREST);
   gl_->TexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_S, GL_CLAMP_TO_EDGE);
   gl_->TexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
-  gl_->TexImage2D(GL_TEXTURE_2D, 0, GL_RGBA8, config.size.width,
-                  config.size.height, 0, GL_RGBA, GL_UNSIGNED_BYTE, nullptr);
+  gl_->TexImage2D(GL_TEXTURE_2D, 0, format_.texture_format, config.size.width,
+                  config.size.height, 0, format_.texture_format,
+                  GL_UNSIGNED_BYTE, nullptr);
   gl_->BindTexture(GL_TEXTURE_2D, 0);
 
-  gl_->FramebufferTexture2D(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0_EXT,
```

Comment:
``GL_COLOR_ATTACHMENT0` is the same value as `GL_COLOR_ATTACHMENT0_EXT` but available in OpenGL ES 2.0. Suggestion by @chinmaygarde